frontend/src/components/Listing/ListingGrid.tsx

```diff
@@ -22,17 +22,61 @@ export interface IListingGridProps {
 }
 
 export default function ListingGrid(props: IListingGridProps) {
+    const [sortingType,setSortingType] = React.useState("A-Z");
     if (props.listings.length === 0) {
         return (
             <Typography variant="h4" align='center' color='text.secondary' mt={2}>
                 No listings found
             </Typography>
         );
     }
+
+    const sortListings = (listings : Array<any>) => {
+        console.log("I was called");
+        const myClonedArray: any[] = [];
+        listings.forEach(val => myClonedArray.push(Object.assign({}, val)));
+        switch (sortingType){
+            case "A-Z":
```

Comment:
maybe replace A-Z stuff with a name that correctly implies that its sorting by title

Reply:
Names replaced with more specific titles